corehq/motech/repeaters/models.py

```diff
@@ -275,9 +275,10 @@ class Repeater(RepeaterSuperProxy):
     is_paused = models.BooleanField(default=False)
     next_attempt_at = models.DateTimeField(null=True, blank=True)
     last_attempt_at = models.DateTimeField(null=True, blank=True)
+    max_workers = models.IntegerField(default=0)
```

Comment:
Any particular reason to default to 0? Given what this will control, I assume defaulting to 1 is also a valid option?

Reply:
Yes. By setting it to 0, we make it easy to manage the default number of workers [using `django.conf.settings`](https://github.com/dimagi/commcare-hq/blob/c28c11b84578ef5613931fa7012278cf438a07a9/corehq/motech/repeaters/models.py#L376).

Defaulting to 1 has pros and cons:
* Pro: All repeat records are sent in chronological order.
* Con: If all sent payloads fail then [the repeater will back off](https://github.com/dimagi/commcare-hq/blob/74137f99d53d70639b48ea9c01e6eefabd80b5e1/corehq/motech/repeaters/tasks.py#L458). If we are only sending one, then every failure postpones the next attempt by an hour.